frontend/src/components/Forms/DevPortfolioForm/DevPortfolioForm.tsx

```diff
@@ -230,6 +241,22 @@ const DevPortfolioForm: React.FC = () => {
   );
 };
 
+const DocumentationInput = ({
+  documentationText,
+  setDocumentationText
+}: {
+  documentationText: string;
+  setDocumentationText: React.Dispatch<React.SetStateAction<string>>;
+}) => (
+  <div>
+    <label className={styles.bold}>
+      Documentation: <span className={styles.red_color}>*</span>
+    </label>
+    <p>Please provide a link to at least one piece of documentation you added/updated.</p>
```

Comment:
Hmm, I'm not sure if it's clearer to either have the developer simply specify and say "documentation located in PR #_" or have them re-paste the link

Reply:
I'd vote for the first one. If it's already included they can let us know. But I'll make it clear that it's OK if they didn't include it in the PRs, and that they can just do a separate documentation PR to fulfill the requirement too.

Reply:
That sounds good!